### ISSUE TITLE
Feature: Expert only mode (i.e., only the text box)

### ISSUE THREAD

**Issue by doktorstick** [0xbs/premade-groups-filter#32]
I've not used anything but the advanced filter text box. The unused full window occupies a lot of real estate. It would be a nice-to-have to only have the advanced filter text box shown and placed below the dungeon finder window. That placement has the secondary effect of not interfering with the default raider.io window. (Or draggable.)

**Comment by 0xbs** [comment]
Would you mind testing version `1.35-beta2` and report any errors?

**Comment by doktorstick** [comment]
Sure. I'll give it a whirl tonight.

**Comment by doktorstick** [comment]
Seems to be working. I'm able to unlock it, move it around. It remembers where it is. Toggling the mode in its location works. My expression is remembered between changing the state as well. I did not test the simpler mode. I also checked Swatter to see if there were some hidden Lua errors and I didn't see any attributed to your mod.

It doesn't use the upper-left as the anchor point when expanding/collapsing, but that's a minor nit and not necessarily wrong; I'll never notice since I leave it in expert mode. It clamps to the top of the screen, which is good for this behavior so the expand/collapse arrow doesn't leave the screen.

It's not ElvUI skinned, so if you work with whoever does that work, you might want to pass along the info once the update is near.

Thanks for doing this. It's really reclaimed some UI space when in DF.
